deployment/copilot/createReplayer.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+
+# Wish list
+# * Allow adding additional security group ids for the Replayer service
+
+# Allow executing this script from any dir
+script_abs_path=$(readlink -f "$0")
+script_dir_abs_path=$(dirname "$script_abs_path")
+cd $script_dir_abs_path
+
+usage() {
+  echo ""
+  echo "Create and Deploy Copilot Replayer service"
+  echo ""
+  echo "Options:"
+  echo "  --id*                                 [string] The unique ID to give this particular Replayer service, will be used in service naming (e.g. traffic-replayer-ID)"
+  echo "  --target-uri*                         [string] The URI of the target cluster that captured requests will be replayed to (e.g. https://my-target-cluster.com:443)"
+  echo "  --kafka-group-id                      [string, default: logging-group-ID] The Kafka consumer group ID that will decide which consumer group the Replayer will join and consume from"
```

Comment:
I the default actually a static string?  I would hope that you would cons up something unique for this replayer, like with the id value.